code/src/components/Search.js

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react'
+import { Link } from 'react-router-dom'
+
+import { SEARCH_API } from '../reusables/urls'
+
+import { MovieCard } from '../components/MovieCard'
+
+export const Search = () => {
+  const [search, setSearch] = useState('')
+  const [id, setId] = useState([])
+  const [error, setError] = useState('')
+
+  const onNewSearch = (event) => {
+    setSearch(event.target.value)
+  }
+
+  const onSearchSubmit = (event) => {
+    event.preventDefault()
+    if (search === '') {
+      setError('Sorry, we couldn´t find any movies that match your search...')
+    } else {
+    fetch(SEARCH_API(search))
+    .then(res => res.json())
+    .then(json => setId(json.results))
+    .catch(err => console.err(err))
```

Comment:
Please remember to remove all console outputs from your code before submitting. 👮🏻

Reply:
@KarolCislowski - We have removed all console outputs now!